components/TechnologiesDetailsModal.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { MoreHorizontal } from 'lucide-react';
+import Image from 'next/image';
+import React, { type FC, ReactElement } from 'react';
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import { Spinner } from '@/components/ui/spinner';
+import { Table, TableBody, TableCell, TableRow } from '@/components/ui/table';
+
+type Technology = {
+  name: string;
+  category: number;
+  icon: string;
+  index: string;
+  sourceUrl: string;
+  detectedTime: number;
+  latestDetectedTime: number;
+  website: string;
+  siteListUrl: string;
+  categoryString?: string;
+};
+
+type TechnologiesDetailsProps = {
+  technologies: Technology[];
+};
+
+const TechnologiesDetailsModal: FC<TechnologiesDetailsProps> = ({
+  technologies,
+}): ReactElement => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <div
+        key={'more'}
+        onClick={() => setOpen(true)}
+        className={`flex h-12 w-12 justify-center rounded-lg hover:cursor-pointer hover:bg-black/5`}
+      >
+        <MoreHorizontal className="m-auto h-5 w-5 text-black" />
```

Comment:
The trigger needs a white text color in the darkmode.

Reply:
![image](https://github.com/maaaathis/digga/assets/37186532/d15e9546-c0e1-4cc9-8c0a-07fe54b6e3d8)

Reply:
I would also find the lucide icon "grip" suitable here, as not only the other technologies are displayed via the modal, but all of them in more detail.